res/values/strings.xml

```diff
@@ -666,8 +684,9 @@
     <string name="autocrypt_incorrect_desktop">Incorrect setup code. Please try again.</string>
     <string name="create_chat_error_desktop">Could not create chat.</string>
     <string name="ask_delete_chat_desktop">Delete this chat?</string>
+    <string name="ask_delete_named_chat_desktop">Are you sure you want to delete \"%1$s\"?</string>
     <string name="email_validation_failed_desktop">Email address required.</string>
-    <string name="forget_login_confirmation_desktop">Delete this login? Everything will be deleted, including your end-to-end setup, contacts, chats, messages and media. This action cannot be undone.</string>
```

Comment:
This string is currently used in AccountManager.java (which confirms @r10s' statement that some `_desktop` strings are used in Android), you should probably change it there to `delete_account_explain_with_name` in the same PR

Reply:
lets keep it for now I haven't setup an android dev environment right now, so it is too much work for me now.
If you want you can do a quick pr after this one, if you agree on the changes I did to the dialog in desktop:
https://github.com/deltachat/deltachat-desktop/pull/1911
![https://user-images.githubusercontent.com/18725968/95263236-c2d19e00-082d-11eb-943d-ab9f12dc6520.png](https://user-images.githubusercontent.com/18725968/95263236-c2d19e00-082d-11eb-943d-ab9f12dc6520.png)